nephio-webui/Kptfile

```diff
@@ -7,4 +7,9 @@ metadata:
 info:
   site: https://nephio.org
   description: Package for the Nephio Web UI.
-pipeline: {}
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/starlark:v0.5.0
+    configPath: set-auth.yaml
+  - image: docker.io/nephio/gen-configmap-fn:latest
```

Comment:
Is it a good idea to use `latest` tag?

Reply:
good point. let's hold on merging this one until that PR merges, then we can tag that image with a different tag

/hold

Reply:
fixed